planning/behavior_path_planner/include/behavior_path_planner/scene_module/pull_over/goal_searcher_base.hpp

```diff
@@ -28,22 +28,26 @@
 namespace behavior_path_planner
 {
 using geometry_msgs::msg::Pose;
+using tier4_autoware_utils::MultiPolygon2d;
 
 struct GoalCandidate
 {
   Pose goal_pose{};
   double distance_from_original_goal{0.0};
   double lateral_offset{0.0};
+  size_t id{0};
 
   bool operator<(const GoalCandidate & other) const noexcept
   {
     // compare in order of decreasing lateral offset.
-    if (std::abs(lateral_offset - other.lateral_offset) > std::numeric_limits<double>::epsilon()) {
-      return lateral_offset < other.lateral_offset;
-    }
+    // if (std::abs(lateral_offset - other.lateral_offset) > std::numeric_limits<double>::epsilon())
+    // {
+    //   return lateral_offset < other.lateral_offset;
+    // }
```

Comment:
Could you remove this comment?

Reply:
removed in https://github.com/autowarefoundation/autoware.universe/pull/2335/commits/20b1e6e9130b0a7c76d4ca2adff87fcbf0572eac